sdk/filter/multifilter.go

```diff
@@ -14,6 +14,8 @@ func NewMultiFilter(filter ...Filter) *MultiFilter {
 	return &MultiFilter{filters: filter}
 }
 
+func (m *MultiFilter) Start() {}
```

Comment:
`MultiFilter.Start` Should call Start for each filter.

Reply:
done